src/config.jl

```diff
@@ -202,7 +202,7 @@ end
 struct OrganisationMember <: Engagement end
 struct IndustryRepresentative <: Engagement end
 struct ScientificExpert <: Engagement end
-struct FishingAssociationRepresentative <: Engagement end
+@with_kw mutable struct FishingAssociationRepresentative <:Engagement interact_WC::Bool = false end
```

Comment:
Once you have a struct with contents, each property should be on a new line:
```julia
@with_kw mutable struct FishingAssociationRepresentative <: Engagement
    interact_WC::Bool = false
end
```

Reply:
Okay, thank you!

Reply:
So you can change that now on your computer, commit the change, and push it again. Once that's done you can click the "Resolve Conversation" button here.